pages/index.page.js

```diff
@@ -18,6 +18,7 @@ import Footer from "../components/layout/Footer";
 import dynamic from "next/dynamic";
 import Head from "next/head";
 import { ButtonLink, Section } from "../components/UI";
+import ScrollButton from "../components/MoveToTop";
```

Comment:
so this way it will only comes in index page.. and what about other pages?

Reply:
resolved this issue